wn_bin target broken with ninja-build

The issue was originally discovered by @aphecetche (on MacOS?). Just verified the issue on my Linux setup:

```
$ git clone https://github.com/FairRootGroup/DDS && cd DDS && mkdir build && cd build
DDS/build$ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=./install ..
-- Building DDS version: 2.1.1.g181b66a
-- Build API docs - YES
-- Boost version: 1.67.0
-- Found the following Boost libraries:
--   log
--   log_setup
--   thread
--   program_options
--   filesystem
--   system
--   regex
--   signals
--   date_time
--   chrono
--   atomic
-- Build MiscCommon - YES
-- Build the dds_sys_files lib - YES
-- Build the dds_ncf lib - YES
-- Build dds_ncf unit tests - NO
-- Build MiscCommon unit tests - NO
-- Build dds_protocol_lib - YES
-- Build dds_protocol_lib tests - NO
-- Build dds-daemonize - YES
-- Build dds-daemonize unit tests - NO
-- Build dds-commander - YES
-- Build dds-commander unit tests - NO
-- Build dds-topology - YES
-- Build dds_topology_lib - YES
-- Build dds_topology_lib unit tests - NO
-- Build dds-agent - YES
-- Build dds-agent unit tests - NO
-- Build dds-user-defaults - YES
-- Build dds-user-defaults unit tests - NO
-- Build dds-info - YES
-- Build dds-submit - YES
-- Build dds-agent-cmd - YES
-- Build dds-test - YES
-- Build dds-octopus - YES
-- Build dds-tutorials - YES
-- Build Tutorial1 - YES
-- Build Tutorial2 - YES
-- Build Tutorial3 - YES
-- Build dds-stat - YES
-- Build dds-custom-cmd - YES
-- Build dds_intercom_lib - YES
-- Build dds_intercom_lib tests - NO
-- Build dds-submit-ssh - YES
-- Build dds-submit-ssh unit tests - NO
-- Build dds-submit-localhost - YES
-- Build dds-submit-slurm - YES
-- Build dds-submit-pbs - YES
-- Build dds-submit-lsf - YES
-- Build dds-session - YES
-- Configuring done
-- Generating done
-- Build files have been written to: /home/dklein/projects/DDS/build
DDS/build$ cmake --build . --target install
(...) works fine
DDS/build$ cmake --build . --target wn_bin 
[1/1] Generate WN binary package
FAILED: CMakeFiles/wn_bin 
cd /home/dklein/projects/DDS/build && -mkdir -p /home/dklein/projects/DDS/build/dds-wrk-bin && /usr/bin/cmake -E copy /home/dklein/projects/DDS/build/dds-agent/dds-agent /home/dklein/projects/DDS/build/dds-wrk-bin && /usr/bin/cmake -E copy /home/dklein/projects/DDS/build/dds-user-defaults/dds-user-defaults /home/dklein/projects/DDS/build/dds-wrk-bin && /usr/bin/cmake -DPREREQ_DESTINATION=/home/dklein/projects/DDS/build/dds-wrk-bin -DDDS_AGENT_BIN_PATH=/home/dklein/projects/DDS/build/dds-agent/dds-agent -DPREREQ_DIRS=/home/dklein/fairsoft_may18/lib::/home/dklein/projects/DDS/build/dds-user-defaults::/home/dklein/projects/DDS/build/dds-protocol-lib::/home/dklein/projects/DDS/build/dds-intercom-lib::/home/dklein/projects/DDS/build/dds-topology-lib::/home/dklein/projects/DDS/build/MiscCommon/dds_ncf:: -DDDS_BOOST_LIB_DIR=/home/dklein/fairsoft_may18/lib -P /home/dklein/projects/DDS/cmake/modules/DDS_CollectPrerequisites.cmake && /usr/bin/cmake -E tar czf dds-wrk-bin-2.1.1.g181b66a-Linux-amd64.tar.gz /home/dklein/projects/DDS/build/dds-wrk-bin && -chmod go+xr dds-wrk-bin-2.1.1.g181b66a-Linux-amd64.tar.gz
/bin/sh: -mkdir: command not found
ninja: build stopped: subcommand failed.
```

Close, if DDS does not support ninja-build.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -454,13 +454,13 @@ endif()
 # TODO: replace DDS_CollectPrerequisites.cmake by DDS_CollectPrerequisitesGen.cmake and make the script more generic
 #
 add_custom_target( wn_bin
-	COMMAND -mkdir -p ${WN_PKG_DIR}
+	COMMAND ${CMAKE_COMMAND} -E make_directory ${WN_PKG_DIR}
 	COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:dds-agent> "${WN_PKG_DIR}"
 	COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:dds-user-defaults> "${WN_PKG_DIR}"
 	COMMAND ${CMAKE_COMMAND} -DPREREQ_DESTINATION=${WN_PKG_DIR} -DDDS_AGENT_BIN_PATH=${DDS_AGENT_BIN_PATH}
 		-DPREREQ_DIRS=${PREREQ_DIRS} -DDDS_BOOST_LIB_DIR=${DDS_BOOST_LIB_DIR} -P "${CMAKE_SOURCE_DIR}/cmake/modules/DDS_CollectPrerequisites.cmake"
 	COMMAND ${CMAKE_COMMAND} -E tar czf ${WN_PKG_NAME_ARCH} "${WN_PKG_DIR}"
-	COMMAND -chmod go+xr ${WN_PKG_NAME_ARCH}
+	COMMAND chmod go+xr ${WN_PKG_NAME_ARCH}
 	COMMENT "Generate WN binary package"
 	VERBATIM )
 

